dist/yaml/ovnkube-master.yaml

```diff
@@ -38,8 +38,13 @@ spec:
       # ovs flow for ovn (geneve)
       # /usr/share/openvswitch/scripts/ovs-ctl --protocol=udp --dport=6081 enable-protocol
       - name: ovnkube-master
-        # this is development build image
-        image: "netdev31:5000/ovn-kube:latest"
+        # NOTE: the daemonset and ovnkube.sh script in the image must be compatible
+        #       This is daemonset version 1 - It starts all master and node daemons in a single container
+        # This is an official build
+        # image: "registry.reg-aws.openshift.com:443/openshift3/ose-ovn-kubernetes:v3.11.0-123483"
+        image: "brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/openshift3/ose-ovn-kubernetes:v3.11.0-123483"
```

Comment:
Can we use the dockerhub account?

Reply:
@rajatchopra The files under the dist directory are the temporary home of OKD specific files. I didn't have a better place to put them. I am not sure how much of what is here is useful to general people in the community. In particular the openshift3/ose-ovn-kubernetes image is built on rhel:7, my development/test images, ovn-kube, are built on centos:7 or fedora:28

Now that we have openshift/ose-ovn-kubernetes we might decide to put the image related files there. I am not sure where the daemonset files belong. We may end up with an operator repo that contains them. Time will tell.

Reply:
@rajatchopra Would it be possible to to have an official docker.io/quay.io repo for release images, images that are built that correspond to releases. We could make a new release, v0.4.0 or some such, and make the official image available for all to use.

At present I have a quay.io/pecameron/ovn-kube repo for my builds, maybe a ovn-kubernetes/ovn-kube repo could be created. What do you think?